satellite/console/consoleweb/utils.go

```diff
@@ -63,6 +67,7 @@ func getQuery(req *http.Request) (query graphqlJSON, err error) {
 
 // queryPOST retrieves graphql query from POST request
 func queryPOST(req *http.Request) (query graphqlJSON, err error) {
+
```

Comment:
I guess this extra new line is not needed

Reply:
yes